### PR TITLE
Use entity ID instead of table uri for Atlas part

### PR DESCRIPTION
**Pull request by L-KID**
### Summary of Changes

Use entity guid instead of table uri. Finished Atlas part. To do FrontEnd and Neo4j.

### Tests

Modified test_atlas_proxy to test using of entity guid.

### CODE REVIEW

**Comment by verdan** [comment]
Nice. few things. 
- Please make sure to revert the config settings. 
- Secondly, I think it would be unfair to use the word `guid`. can we please rename that to only id? `table_id`, `column_id` ? 
- Last, I think Neo4j does not use id for their operations, as they are not persistent. (not 100% sure, need to check). But, we should make this change incrementally otherwise, to make it work for both neo4j and atlas.

File: tests/unit/proxy/test_atlas_proxy.py [hunk 1]
```diff
@@ -47,7 +48,7 @@ def setUp(self):
         }
 
         self.db_entity = {
-            'guid': '-100',
+            'guid': 'ff6f8097-aead-428a-94ca-3fcb92b02255',
```
Comment (verdan) [inline tests/unit/proxy/test_atlas_proxy.py:51]: can we please revert it and make it simple integer ? That would be really easy to test and compare.

**Review by verdan — CHANGES_REQUESTED** [review]
@L-KID nice work!! we still need to think of neo4j proxy.

File: tests/unit/proxy/test_atlas_proxy.py (hunk 1 above)
Comment (L-KID) [reply tests/unit/proxy/test_atlas_proxy.py:51]: Okay, of course. But I'm also thinking to make it a bit more real using those mock guid.

**Pull request merged**